src/documents/ChangeApplierAPI.md

```diff
@@ -60,14 +60,90 @@ A `<modelPathReference>` has the form:
+a path specification.
+
+<table>
+    <thead>
+        <tr>
+            <th colspan="3">Possible values for the <code>path</code> member of a model listener declaration</th>
+        </tr>
+        <tr>
+            <th>Type</th>
+            <th>Meaning</th>
+            <th>Examples</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>String</code></td>
+            <td>A <code>&lt;shortModelPathReference&gt;</code></td>
+            <td>
+                See examples in section [Model Path References](#model-path-references)
+            </td>
+        </tr>
+        <tr>
+            <td><code>Object</code></td>
+            <td>A `<modelPathRecord>`, including members <ul>
+                <li><code>segs</code> (<code>Array</code> - required) and</li> 
+                <li><code>context</code> (<code>String</code> - optional) which encodes the model path to be matched.</li>
+                </ul>
+                </td>
+            <td>These examples encode exactly the same path expressions, in the same order as in the section [Model Path References](#model-path-references) 
+                <ul>
+                    <li><code>{segs: ["modelPath"]}</code></li>
+                    <li><code>{segs: ["modelPath", "&#42;"]}</code></li>
+                    <li><code>{segs: []}</code></li>
+                    <li><code>{segs: ["&#42;"]}</code></li>
+                    <li><code>{segs: ["modelPath"], context: "otherComponent"}</code></li>
+                    <li><code>{segs: ["modelPath", "&#42;"], context: "otherComponent"}</code></li>
+                    <li><code>{segs: [], context: "otherComponent"}</code></li>
+                    <li><code>{segs: ["&#42;"], context: "otherComponent"}</code></li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <td><code>Array</code></td>
+            <td>An array each of whose element may either be a String holding a <code>&lt;shortModelPathReference&gt;</code> or a <code>&lt;modelPathRecord&gt;</code>. The listener will be notified when <it>any</it> of these
+            paths receive changes. Below is further information on <a href="#matching-on-multiple-paths-in-a-single-model-listener-declaration"><code>matching on multiple paths</code></a></td>
+            <td><code>[
+                    "position", {
+                        segs: [["windowHolders"], "{that}.options.ourWindow"]
+                    }
+                ]</code>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+#### Matching on multiple paths in a single model listener declaration
+
+When the `path` member of a model listener declaration holds an `Array`, the listener will be notified when <it>any</it> of these paths receive changes. 
 
+Note that a listener which specifies references to multiple component targets in such a list will just receive one notification per component of its changes. A listener which supplies an
```

Comment:
Can you clarify "just receive one notification per component of its changes". 

Just a guess, does it mean when one listener is listening to multiple model paths like "x" and "x.y". When a change is made on "x.y" that trigger model changes on both "x" and "x.y", the listener only receives one notification rather than two?